oracle/README.md

```diff
@@ -2,17 +2,15 @@
 
 ## Overview
 
-Get metrics from oracle service in real time to:
-
-* Visualize and monitor oracle states
-* Be notified about oracle failovers and events.
+Get metrics from Oracle Database servers in real time to:
+* Visualize and monitor your Oracle Database's availability and performance metrics.
```

Comment:
We should just merge this into one sentence. It doesn't make sense to have a single bullet point.
`Get metrics from Oracle Database servers in real time to visualize and monitor availability and performance metrics.`